src/index.ts

```diff
@@ -47,20 +51,26 @@ export async function startNotarize(opts: NotarizeStartOptions): Promise<Notariz
     }
     d('zip succeeded, attempting to upload to apple');
 
+    const notarizeArgs = [
+      'altool',
+      '--notarize-app',
+      '-f',
+      zipPath,
+      '--primary-bundle-id',
+      opts.appBundleId,
+      '-u',
+      makeSecret(opts.appleId),
+      '-p',
+      makeSecret(opts.appleIdPassword),
+    ];
+
+    if (opts.ascProvider) {
+      notarizeStartOpts.push('-itc_provider', opts.ascProvider);
+    }
+
     const result = await spawn(
       'xcrun',
-      [
-        'altool',
-        '--notarize-app',
-        '-f',
-        zipPath,
-        '--primary-bundle-id',
-        opts.appBundleId,
-        '-u',
-        makeSecret(opts.appleId),
-        '-p',
-        makeSecret(opts.appleIdPassword),
-      ],
+      notarizeStartOpts,
```

Comment:
This can't be working unless I'm missing something, the variable is called `notarizeArgs` above?

Reply:
Oh yikes. I forgot to change that when I made the change for https://github.com/electron-userland/electron-notarize/pull/1#discussion_r238084835